easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-4.1-foss-2017b.eb

```diff
@@ -0,0 +1,38 @@
+name = 'OpenFOAM'
+version = '4.1'
+
+homepage = 'http://www.openfoam.com/'
+
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/OpenFOAM/OpenFOAM-%(version_major)s.x/archive']
+sources = ['version-%(version)s.tar.gz']
+patches = ['OpenFOAM-%(version)s-cleanup.patch']
+checksums = [
+    '2de18de64e7abdb1b649ad8e9d2d58b77a2b188fb5bcb6f7c2a038282081fd31',  # version-4.1.tar.gz
+    '7a9d418f18c475fb48ddc427be7271b0fd759ad183f09a6bcafe4b76924b2a70',  # OpenFOAM-4.1-cleanup.patch
+]
+
+dependencies = [
+    ('ParaView', '5.2.0', '-mpi'),
```

Comment:
this is quite old, why not use latest 5.4.1?

Reply:
openFOAM build fails with paraview 5.4.1;
in the release notes of OpenFOAM 5 you can find "updated paraview version to 5.4
i assumed you need older version of paraview

Reply:
I run 2 tests on OpenFOAM 5 build with this CGAL i did not find anything strange,

Reply:
OK, thanks for clarifying.